src/index.html

```diff
@@ -6,9 +6,39 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+      crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
+
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href ="#home" class="logo">
+        <img
+          src="images/logo.png"
+          alt="logo"
+          class="logo_img">
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item"><a href="#apple" class="nav__link is-active">APPLE</a></li>
+          <li class="nav__item"><a href="#samsung" class="nav__link">SAMSUNG</a></li>
+          <li class="nav__item"><a href="#smartphones" class="nav__link">SMARTPHONES</a></li>
+          <li class="nav__item">
+            <a href="#laptops&computers"
+              class="nav__link"
+              data-qa="hover">LAPTOPS & COMPUTERS
+            </a>
```

Comment:
don't forget about code style
```suggestion
            <a href="#laptops&computers"
              class="nav__link"
              data-qa="hover"
            >
              LAPTOPS & COMPUTERS
            </a>
```